assembl/static2/package.json

```diff
@@ -146,6 +148,17 @@
   },
   "//": "See https://facebook.github.io/jest/docs/webpack.html for more info",
   "jest": {
+    "coverageReporters": [
+      "json-summary"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "lines": 53.04,
```

Comment:
current values! we need to update them! we have to update them manually!

Reply:
No, it's done automatically with jest-coverage-ratchet